nbs/src/core/models.ipynb

```diff
@@ -42,7 +42,7 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #52.        def __repr__(self):
</pre><p>We don't need to override this method</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='388'/>